_pages/hellpheonix.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: hellpheonix
+pageUrl: hellpheonix.github.io
+timestamp: 2018-01-11
+---
```

Comment:
@hellpheonix add a line break at the end here

Reply:
@hectorsector I have added a line break, thank you for remingding

Reply:
@hellofisher our automated tests check for any pages that may no longer be active. It appears that [one wasn't](https://travis-ci.org/githubschool/on-demand-github-pages/builds/328616318#L2299). That's why your tests were failing. I've removed the nonfunctional page from your branch it should now work.